packages/framework/build.xml

```diff
@@ -634,9 +640,10 @@
         <if>
             <not><isset property="environment"/></not>
             <then>
-                <exec command="php bin/console about | grep Environment | sed 's/Environment//;s/ //g'" outputProperty="environment"/>
+                <exec command="php ${path.app}/getEnvironment.php" outputProperty="environment"/>
 
                 <property name="environment" value="${environment}"/>
+                <echo>${environment}</echo>
```

Comment:
not sure whether this is necessary, it means that the environment info is now printed every time any target with protection is executed, but on the other hand, why not, I do not mind it :grin: